0x01-de.sbv

```diff
@@ -0,0 +1,969 @@
+sollte in etwa so aussehen.
+
+0:20:09.040,0:20:11.600
+Es ist wichtig ihren Token aufzuschreiben.
+
+0:20:12.400,0:20:14.720
+Wir können doppelt prüfen 
+ob die Registrierung 
+
+0:20:14.720,0:20:18.000
+erfolgreich war, 
+indem wir auf die Historienseite gehen
+
+0:20:18.000,0:20:22.000
+und dort unsere XMR-Adresse und 
+Tokens eingeben, das wir erhalten haben.
+
+0:20:22.960,0:20:27.514
+Wenn Sie Probleme mit diesem Schritt haben,
+gehen Sie zum xmrvsbeast Subreddit
+
+0:20:27.600,0:20:31.520
+ und suchen Sie nach ähnlichen
+ Problemen, bevor Sie dort etwas posten.
+
+0:20:32.240,0:20:34.640
+Ein Link dazu ist in der Beschreibung zu finden.
+
+0:20:41.280,0:20:46.880
+Ihren Fortschritt aus der ferne können sie 
+leicht auf https://p2pool.observer/ überprüfen.
+
+0:20:47.440,0:20:52.160
+Hier können mit Ihre Monero-Adresse 
+Ihren individuellen Fortschritt finden.
+
+0:20:53.276,0:20:56.156
+Da wir neu sind, bekommen wir eine Fehlermeldung.
+
+0:20:56.560,0:21:02.480
+Erst wenn Sie Anteile gefunden wurden sehen Sie 
+Informationen über Ihren Miner und seinen Fortschritt.
+
+0:21:03.120,0:21:05.920
+Um herauszufinden, wie oft Sie Anteile produzieren,
+
+0:21:05.920,0:21:10.880
+gehen Sie zurück zur Hauptseite, wählen Sie den
+Rechner für die durchschnittliche Produktionszeit,
+
+0:21:10.880,0:21:14.000
+geben Sie Ihre Hashrate ein und 
+Sie erhalten eine Schätzung.
+
+0:21:14.880,0:21:18.400
+Werfen wir einen Blick auf einen anderen 
+Miner-Fortschritt, um ein Beispiel zu sehen.
+
+0:21:19.680,0:21:24.160
+Hier sehen Sie die Statistiken des miners,
+einschließlich des durchschnittlichen Aufwands,
+
+0:21:24.160,0:21:26.960
+gefundene Blöcke und Anteilspositionen.
+
+0:21:26.960,0:21:32.240
+Anteile bewegen sich von rechts nach links 
+mit der Zeit im Gegensatz zum p2pool-Programm.
+
+0:21:32.240,0:21:35.440
+Darunter sehen Sie 
+Informationen über jede Auszahlung  
+
+0:21:35.440,0:21:38.916
+und die Aktien, die diese Auszahlungen 
+diese Auszahlungen ermöglicht haben.
+
+0:21:43.120,0:21:47.520
+Dieser Pool ist nur möglich durch 
+durch die harte Arbeit der Spender.  
+
+0:21:47.520,0:21:51.760
+Das Hauptrepository für p2pool finden Sie 
+ist in der Beschreibung unten zu finden.
+
+0:21:52.320,0:21:56.080
+Wenn Sie mehr wissen wollen wie es 
+funktioniert und welche Funktionen es hat,  
+
+0:21:56.080,0:21:59.680
+und wie man es auf einem 
+anderen Betriebssystem einrichtet,
+
+0:21:59.680,0:22:01.156
+hier ist der richtige Ort.
+
+0:22:01.600,0:22:05.440
+Und unten auf dieser Seite 
+finden Sie eine Spendenadresse.
+
+0:22:05.440,0:22:09.360
+Die Menschen, die an diesen Projekte arbeiten, 
+tun es hauptsächlich aus Liebe, nicht für Geld,  
+
+0:22:09.360,0:22:12.480
+Jede Spende dient also dazu, sie zu motivieren.
+
+0:22:13.440,0:22:18.356
+Nun, das war's mit diesem Video. Vielen Dank 
+fürs Zuschauen und viel Spaß beim Mining.
+
+0:22:18.462,0:22:20.369
+~moneroguides
+
+0:22:20.462,0:22:23.369
+Deutsche übersetzung:
```

Comment:
"German Translation :ZeoZ"
if that happen to bother you, you can just delete it

Reply:
not at all, you deserve the credits